middleware/packet-forward-middleware/docs/integration.md

```diff
@@ -0,0 +1,156 @@
+# Integration
+This document provides instructions on integrating and configuring the Packet Forward Middleware (PFM) within your
+existing chain implementation. This document is _NOT_ a guide on developing with the Cosmos SDK or ibc-go and makes
+the assumption that you have some existing codebase for your chain with IBC already enabled.
```

Comment:
Useful to do a tl;dr up front perhaps? 

For example: 
1. Set up the PFM module + Keeper, and add logic into Begin + End Block 
2. Configure the transfer stack to use the middleware
3. Set additional configuration options (e.g. ...)

Reply:
Makes sense, I will get that included here as well!